src/core/spectrum.cpp

```diff
@@ -13,9 +13,9 @@ void spectrum_from_file(const fs::path &path, std::vector<Scalar> &wavelengths,
     auto fs = Thread::thread()->file_resolver();
     fs::path file_path = fs->resolve(path);
     if (!fs::exists(file_path))
-        Log(Error, "\"%s\": file does not exist!", file_path);
+        Log(Error, "\"%s\": file does not exist!", file_path.string());
 
-    Log(Info, "Loading spectral data file \"%s\" ..", file_path);
+    Log(Info, "Loading spectral data file \"%s\" ..", file_path.string());
```

Comment:
Does this fix a specific bug? What is the log message before and after this change?

Reply:
I actually made these types of changes during a very pedantic file path review for consistency in explicitly calling `.string()`; I don't know that it fixes a bug. I will back out this for less noise in the PR!